document/src/test/resources/config/application.yml

```diff
@@ -8,8 +8,8 @@ spring:
         hikari:
             auto-commit: false
     liquibase:
-        enabled: true
-        change-log: classpath:/config/liquibase/document-master.xml
+        enabled: false
+#        change-log: classpath:/config/liquibase/document-master.xml
```

Comment:
Won't this break the tests?

Reply:
Nope, tests have been fixed. The problem with how it was set up here is none of the liquibase changesets from other modules would run out of the box. This is fine if all we're testing is just document itself, but we aren't when including authorization calls.